tests/Portability/ConnectionTest.php

```diff
@@ -18,7 +18,7 @@ public function testGetServerVersion(): void
             ->method('getServerVersion')
             ->willReturn('1.2.3');
 
-        $connection = new Connection($driverConnection, new Converter(false, false, 0));
+        $connection = new Connection($driverConnection, new Converter(false, false, Converter::CASE_LOWER));
```

Comment:
Wait, why did you change these tests? That shouldn't be necessary, should it?

Reply:
No, this change is not strictly required, as the previous value (0) is identical to the value provided by this constant. I made these replacements in order to be consistent with the usage of these constants, as I think this also help as an example.
I can revert these changes if you think they are not a proper move for this PR.

Reply:
No, you're right. It's better this way, thanks for the explanation.